index.html

```diff
@@ -14,7 +14,7 @@
 		}
         body {
             font-family: Arial;
-            background-color: #AD8E58;
+            background-color: #5C6BC0;
```

Comment:
```suggestion
            background-color: #6E9E6D;
```

Reply:
Wow, love the design!

Reply:
I made this pull request when I was 15 🤣. I'm glad it finally got accepted.